index.js

```diff
@@ -0,0 +1,75 @@
+/* checking the local storage for a book collection list. If non present,
+we create a new array to store new books. */
+
+let bookCollection = JSON.parse(localStorage.getItem('bookCollection')) || [];
+
+// Getting documents from html
+
+const bookCollectionList = document.querySelector('.book-list');
+const addBtn = document.getElementById('add-form');
+const bookTitle = document.querySelector('.titleinput');
+const bookAuthor = document.querySelector('.authorinput');
+
+// Function to add a new book to the collection
+
+function add(title, author) {
+  bookCollection.push({ title, author });
+  localStorage.setItem('bookCollection', JSON.stringify(bookCollection));
+}
+
+// Function to remove book from collection
+
+function remove(index) {
+  bookCollection = bookCollection.filter((book, bookIndex) => bookIndex !== +index);
+  localStorage.setItem('bookCollection', JSON.stringify(bookCollection));
+}
```

Comment:
**[OPTIONAL]**

Consider using array methods that don't mutate the array, e.g. [concat](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/concat), [filter](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/filter), [slice](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/slice) instead of [push](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/push), [pop](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/pop), [splice](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/splice) and so on.